API_url_tracker.Rmd

```diff
@@ -0,0 +1,60 @@
+---
+title: "API URL tracker"
+author: "Oluwasegun Apejoye"
+date: "2023-09-04"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+# install api branch from experience dashboard
+# remotes::install_github("CDU-data-science-team/experiencesdashboard@latest_api_interface")
+# library(experiencesdashboard)
+
+# temporarily import the required function from the experiencesdashboard package
+source("https://raw.githubusercontent.com/CDU-data-science-team/experiencesdashboard/latest_api_interface/R/fct_api_pred.R")
+
+library(dplyr)
+```
+
+# Last rendered at `r round(Sys.time())`
+
+## load the api job table from the database
+
+```{r cars}
+pool <- odbc::dbConnect(
```

Comment:
I think experience has told us not to call connections `pool` because then we all start thinking that it's a `pool` connection which it isn't in this case